controllers/token_scorer/token_scorer.py

```diff
@@ -236,10 +283,43 @@ def process_token_locations(self) -> None:
             )
             self._token_statuses[token] = 0
 
+    def get_scores(self) -> Dict[Owner, int]:
+        """
+        Get the current scores.
+
+        The returned dict will always include all the claimants as keys.
+        """
+        zone_scores: Dict[Owner, int] = {}
+
+        for zone in Owner.zones():
+            zone_scores[zone] = sum(
+                value
+                for token, value in self._token_statuses.items()
+                if token.owner == zone
+            )
+
+        return zone_scores
+
+    def update_displayed_scores(self) -> None:
+        scores = self.get_scores()
+
+        for zone, score in scores.items():
+            display = self.score_displays[zone]
+
+            score_low = score % 10
+            score_high = int(score / 10)
```

Comment:
it's not really low and high, it's more tens and units?

Reply:
e3c0da681beb9f5cc7bdc5f10a4a12a69d6a7b98